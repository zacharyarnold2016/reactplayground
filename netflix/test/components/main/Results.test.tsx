// @ts-nocheck
import { expect } from "@jest/globals";
import renderer from "react-test-renderer";
import Result from "../../../src/components/main/Results";
import { useLoaderData } from "react-router";

useLoaderData = jest.fn(() => 4);

describe("Result", () => {
  test("Snapshot", () => {
    const component = renderer.create(<Result />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
