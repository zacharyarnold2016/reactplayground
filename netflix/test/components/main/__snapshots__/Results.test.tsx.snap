// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Result Snapshot 1`] = `
<div
  className="searchResults"
>
  <h2>
     
     movie found
  </h2>
</div>
`;
